playwright/test/concept.test.ts

```diff
@@ -151,3 +156,106 @@ test.describe('concepts', () => {
     );
   });
 });
+
+const allRecordsLink = async (
+  page: Page,
+  tabId: string | undefined,
+  recordType: string
+) => {
+  // Note: the `link-reset` class is added by ButtonSolid, and is a way to
+  // make sure we find the "All Works" link, and not a link to an individual work.
+  const allRecords = await page.waitForSelector(
+    `div#tabpanel-${tabId} a.link-reset`
```

Comment:
Gareth sent Adi and I [this documentation for selectors best practices](https://testing-library.com/docs/queries/about/#priority) when we created Search, which I found super useful!

As a last resort we'll use `data-testId` (or `data-test-id`? @davidpmccormick which one worked again?)(which has a cool `getByTestId()` helper function), but we really try not to use CSS selectors moving forward so we know that when we're styling/cleaning up, it's really just styling.